crates/teloxide-core/src/types/story.rs

```diff
@@ -0,0 +1,5 @@
+use serde::{Deserialize, Serialize};
+
+/// TBA 6.8: currently it holds no information
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct Story;
```

Comment:
did you check that it works? iirc you need to use a normal structure, not a unit structure (because of a serde quirk)
```suggestion
pub struct Story {}
```